examples/crm/src/dashboard/DealsPipeline.tsx

```diff
@@ -30,11 +31,11 @@ export const DealsPipeline = () => {
             return;
         }
         const deals: Deal[] = [];
-        stages
-            .filter(stage => stage !== 'won')
+        dealStages
+            ?.filter(stage => stage.value !== 'won')
```

Comment:
`?` should not be necessary. Also, the `won` value should be customizable. Something like `dealsPipelineStatuses`